src/test/javascript/spec/service/user.service.spec.ts

```diff
@@ -14,7 +11,7 @@ describe('User Service', () => {
     beforeEach(() => {
         TestBed.configureTestingModule({
             imports: [HttpClientTestingModule],
-            providers: [JhiDateUtils],
+            providers: [],
```

Comment:
Can this line be removed?

Reply:
Apparently this does work (as opposed to removing the import)